elao.app/.manala/ansible/inventories/system.yaml.tmpl

```diff
@@ -150,6 +150,10 @@ system:
                 # App
                 {{- .apt.packages | toYaml | nindent 16 }}
             {{- end }}
+            # Network
+            system_network: true
```

Comment:
`system_network` flag should be dynamic (do we have some network hosts, or not ?)